app/retail/emails.py

```diff
@@ -818,10 +906,13 @@ def render_new_bounty_roundup(to_email):
 
     return response_html, response_txt, subject
 
+# DJANGO REQUESTS
 
 
-
-# DJANGO REQUESTS
+@staff_member_required
+def weekly_recap(request):
+    response_html, _ = render_weekly_recap("mark.beacom@consensys.net")
```

Comment:
@kuhnchris are we leaving this hardcoded here ?

Reply:
This is just for /_administration/, we can put in someone else if you want, we just need someone who had regulary bounty activities so it outputs "something".